content/articles/How-to-build-a-Chrome-Extension-that-displays-motivational-quotes-using-Javascript/index.md

```diff
@@ -0,0 +1,339 @@
+
+To interact with the notifications you can set the field as true. False is okay as we want the notifications to come and go on their own.
+
+Next, we use `chrome.notifications.create` to create the notifications and call the object `options` as a parameter.
+
+Adding this to `the background.js` file we get:
+
+```javascript
+chrome.runtime.onInstalled.addListener(() => {
+  console.log('onInstalled...');
+
+  // create alarm after extension is installed / upgraded
+  chrome.alarms.create('startRequest', { periodInMinutes: 4 });
+  startRequest();
+ 
+  });
+  
+
+  chrome.alarms.onAlarm.addListener(alarm=>{
+    startRequest();
+  });
+
+async function startRequest() {
+  
+  const response = await fetch('https://api.quotable.io/random');
+  const newData = await response.json();
+  const data = `${newData.content} —${newData.author}`
+  console.log(data);
+  // chrome.storage.sync.set({data});
+
+  
+  var options = {
+    title: 'Random Quotes',
+    message: data,
+    iconUrl: '/images/favicon-16x16.png',
+    type: 'basic',
+    // requireInteraction: true
+  }
+  chrome.notifications.create('', options);
+ 
+}
+```
+
+Finally, our motivational quotes are displayed in the notifications box like so:
+
+![working-extension](/engineering-education/How-to-build-a-Chrome-Extension-that-displays-motivational-quotes-using-Javascript/images/completed-notification.PNG)
+
+Now that you have your extension up and running, you may want to publish it on the [Chrome web store](https://developer.chrome.com/docs/webstore/publish/) - you are however required to pay a small fee.
+
+If you decide otherwise you can still share with a few friends, even without publishing on the web-store.
+
+#### What to do?
+- Right-click on the folder containing your extension.
+- select `send to compressed(zip)folder` to create a zip file copy.
+- Share via any medium to your friends e.g mail, hard drives, etc.
+- They can install the extension on their local machines by unzipping the file, then going to the chrome extension management page on their browser and clicking on `load unpacked` as we did in the beginning.  This should prompt them to select the unzipped file folder. Tadaa! it works.
+- Lastly, they should have their system notifications visible.
+
+### Conclusion
+Hooray🎉🎉 you have learned how to build a simple Chrome browser extension, register components like background script in the manifest file, fetch data from APIs, and learned to use Chrome extension APIs like chrome.alarms and chrome.notifications.
+
+You can now share with friends with or without hosting in the web-store.
+
+I'm excited to see the amazing things you'll build 🚀🚀
+
+### References
+- https://developer.chrome.com/docs/extensions/mv3/getstarted/
+- https://www.youtube.com/watch?v=7Tu2j2pc87I&t=418s
+- https://www.youtube.com/watch?v=ew9ut7ixIlI&t=1s
```

Comment:
Please name your references. for example, `Getting started - Chrome Developers`, etc